[sfml] Build error

Package: sfml:arm64-osx -> 2.6.0#17

**Host Environment**

- Host: arm64-osx
- Compiler: AppleClang 15.0.0.15000034 (Sonoma beta 3)
-    vcpkg-tool version: 2023-06-22-f19f3d9939100085fe2f7e41502d60b08e2083b6
    vcpkg-scripts version: a293ce589 2023-06-28 (2 weeks ago)

**To Reproduce**

`vcpkg install `

It also seems like despite setting `-DCMAKE_C_COMPILER:FILEPATH=/opt/homebrew/opt/llvm/bin/clang`, SFML insists on building with the default system clang.

**Failure logs**

```
-- Using cached SFML-SFML-2.6.0.tar.gz.
-- Cleaning sources at /Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source /Users/pierre/Code/VectorAudio/vcpkg/downloads/SFML-SFML-2.6.0.tar.gz
-- Applying patch fix-dependencies.patch
-- Using source at /Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean
-- Configuring arm64-osx
CMake Warning at /Users/pierre/Code/VectorAudio/build/vcpkg_installed/arm64-osx/share/vcpkg-cmake/vcpkg_cmake_configure.cmake:318 (message):
  The following variables are not used in CMakeLists.txt:

      SFML_MISC_INSTALL_PREFIX

  Please recheck them and remove the unnecessary options from the
  `vcpkg_cmake_configure` call.

  If these options should still be passed for whatever reason, please use the
  `MAYBE_UNUSED_VARIABLES` argument.
Call Stack (most recent call first):
  ports/sfml/portfile.cmake:17 (vcpkg_cmake_configure)
  scripts/ports.cmake:147 (include)


-- Building arm64-osx-dbg
CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:134 (message):
    Command failed: /opt/homebrew/Cellar/cmake/3.26.4/bin/cmake --build . --config Debug --target install -- -v -j9
    Working Directory: /Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/arm64-osx-dbg
    See logs for more information:
      /Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/install-arm64-osx-dbg-out.log

Call Stack (most recent call first):
  /Users/pierre/Code/VectorAudio/build/vcpkg_installed/arm64-osx/share/vcpkg-cmake/vcpkg_cmake_build.cmake:74 (vcpkg_execute_build_process)
  /Users/pierre/Code/VectorAudio/build/vcpkg_installed/arm64-osx/share/vcpkg-cmake/vcpkg_cmake_install.cmake:16 (vcpkg_cmake_build)
  ports/sfml/portfile.cmake:26 (vcpkg_cmake_install)
  scripts/ports.cmake:147 (include)



```
<details><summary>/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/install-arm64-osx-dbg-out.log</summary>

```
[1/124] /Library/Developer/CommandLineTools/usr/bin/c++ -DSFML_STATIC -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/include -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/src -fPIC -g -arch arm64 -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX14.0.sdk -fvisibility=hidden -stdlib=libc++  -Wall -Wextra -Wshadow -Wnon-virtual-dtor -Wcast-align -Wunused -Woverloaded-virtual -Wconversion -Wsign-conversion -Wdouble-promotion -Wformat=2 -Wnull-dereference -Wold-style-cast -Wpedantic -Werror -Wno-unknown-warning-option -MD -MT src/SFML/System/CMakeFiles/sfml-system.dir/Sleep.cpp.o -MF src/SFML/System/CMakeFiles/sfml-system.dir/Sleep.cpp.o.d -o src/SFML/System/CMakeFiles/sfml-system.dir/Sleep.cpp.o -c /Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/src/SFML/System/Sleep.cpp
[2/124] /Library/Developer/CommandLineTools/usr/bin/c++ -DSFML_STATIC -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/include -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/src -fPIC -g -arch arm64 -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX14.0.sdk -fvisibility=hidden -stdlib=libc++  -Wall -Wextra -Wshadow -Wnon-virtual-dtor -Wcast-align -Wunused -Woverloaded-virtual -Wconversion -Wsign-conversion -Wdouble-promotion -Wformat=2 -Wnull-dereference -Wold-style-cast -Wpedantic -Werror -Wno-unknown-warning-option -MD -MT src/SFML/System/CMakeFiles/sfml-system.dir/Clock.cpp.o -MF src/SFML/System/CMakeFiles/sfml-system.dir/Clock.cpp.o.d -o src/SFML/System/CMakeFiles/sfml-system.dir/Clock.cpp.o -c /Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/src/SFML/System/Clock.cpp
[3/124] /Library/Developer/CommandLineTools/usr/bin/c++ -DSFML_STATIC -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/include -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/src -fPIC -g -arch arm64 -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX14.0.sdk -fvisibility=hidden -stdlib=libc++  -Wall -Wextra -Wshadow -Wnon-virtual-dtor -Wcast-align -Wunused -Woverloaded-virtual -Wconversion -Wsign-conversion -Wdouble-promotion -Wformat=2 -Wnull-dereference -Wold-style-cast -Wpedantic -Werror -Wno-unknown-warning-option -MD -MT src/SFML/System/CMakeFiles/sfml-system.dir/Lock.cpp.o -MF src/SFML/System/CMakeFiles/sfml-system.dir/Lock.cpp.o.d -o src/SFML/System/CMakeFiles/sfml-system.dir/Lock.cpp.o -c /Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/src/SFML/System/Lock.cpp
[4/124] /Library/Developer/CommandLineTools/usr/bin/c++ -DSFML_STATIC -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/include -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/src -fPIC -g -arch arm64 -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX14.0.sdk -fvisibility=hidden -stdlib=libc++  -Wall -Wextra -Wshadow -Wnon-virtual-dtor -Wcast-align -Wunused -Woverloaded-virtual -Wconversion -Wsign-conversion -Wdouble-promotion -Wformat=2 -Wnull-dereference -Wold-style-cast -Wpedantic -Werror -Wno-unknown-warning-option -MD -MT src/SFML/System/CMakeFiles/sfml-system.dir/Mutex.cpp.o -MF src/SFML/System/CMakeFiles/sfml-system.dir/Mutex.cpp.o.d -o src/SFML/System/CMakeFiles/sfml-system.dir/Mutex.cpp.o -c /Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/src/SFML/System/Mutex.cpp
...
Skipped 5 lines
...

[10/124] /Library/Developer/CommandLineTools/usr/bin/c++ -DSFML_STATIC -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/include -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/src -fPIC -g -arch arm64 -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX14.0.sdk -fvisibility=hidden -stdlib=libc++  -Wall -Wextra -Wshadow -Wnon-virtual-dtor -Wcast-align -Wunused -Woverloaded-virtual -Wconversion -Wsign-conversion -Wdouble-promotion -Wformat=2 -Wnull-dereference -Wold-style-cast -Wpedantic -Werror -Wno-unknown-warning-option -MD -MT src/SFML/System/CMakeFiles/sfml-system.dir/Unix/MutexImpl.cpp.o -MF src/SFML/System/CMakeFiles/sfml-system.dir/Unix/MutexImpl.cpp.o.d -o src/SFML/System/CMakeFiles/sfml-system.dir/Unix/MutexImpl.cpp.o -c /Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/src/SFML/System/Unix/MutexImpl.cpp
[11/124] /Library/Developer/CommandLineTools/usr/bin/c++ -DSFML_STATIC -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/include -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/src -fPIC -g -arch arm64 -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX14.0.sdk -fvisibility=hidden -stdlib=libc++  -Wall -Wextra -Wshadow -Wnon-virtual-dtor -Wcast-align -Wunused -Woverloaded-virtual -Wconversion -Wsign-conversion -Wdouble-promotion -Wformat=2 -Wnull-dereference -Wold-style-cast -Wpedantic -Werror -Wno-unknown-warning-option -MD -MT src/SFML/System/CMakeFiles/sfml-system.dir/Unix/ClockImpl.cpp.o -MF src/SFML/System/CMakeFiles/sfml-system.dir/Unix/ClockImpl.cpp.o.d -o src/SFML/System/CMakeFiles/sfml-system.dir/Unix/ClockImpl.cpp.o -c /Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/src/SFML/System/Unix/ClockImpl.cpp
[12/124] /Library/Developer/CommandLineTools/usr/bin/c++ -DSFML_STATIC -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/include -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/src -fPIC -g -arch arm64 -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX14.0.sdk -fvisibility=hidden -stdlib=libc++  -Wall -Wextra -Wshadow -Wnon-virtual-dtor -Wcast-align -Wunused -Woverloaded-virtual -Wconversion -Wsign-conversion -Wdouble-promotion -Wformat=2 -Wnull-dereference -Wold-style-cast -Wpedantic -Werror -Wno-unknown-warning-option -MD -MT src/SFML/System/CMakeFiles/sfml-system.dir/Unix/ThreadLocalImpl.cpp.o -MF src/SFML/System/CMakeFiles/sfml-system.dir/Unix/ThreadLocalImpl.cpp.o.d -o src/SFML/System/CMakeFiles/sfml-system.dir/Unix/ThreadLocalImpl.cpp.o -c /Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/src/SFML/System/Unix/ThreadLocalImpl.cpp
[13/124] /Library/Developer/CommandLineTools/usr/bin/c++ -DSFML_STATIC -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/include -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/src -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/extlibs/headers/vulkan -isystem /Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/extlibs/headers/glad/include -iframework /Library/Developer/CommandLineTools/SDKs/MacOSX14.0.sdk/System/Library/Frameworks -fPIC -g -arch arm64 -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX14.0.sdk -fvisibility=hidden -stdlib=libc++  -Wall -Wextra -Wshadow -Wnon-virtual-dtor -Wcast-align -Wunused -Woverloaded-virtual -Wconversion -Wsign-conversion -Wdouble-promotion -Wformat=2 -Wnull-dereference -Wold-style-cast -Wpedantic -Werror -Wno-unknown-warning-option -MD -MT src/SFML/Window/CMakeFiles/sfml-window.dir/Context.cpp.o -MF src/SFML/Window/CMakeFiles/sfml-window.dir/Context.cpp.o.d -o src/SFML/Window/CMakeFiles/sfml-window.dir/Context.cpp.o -c /Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/src/SFML/Window/Context.cpp
[14/124] /Library/Developer/CommandLineTools/usr/bin/c++ -DSFML_STATIC -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/include -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/src -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/extlibs/headers/vulkan -isystem /Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/extlibs/headers/glad/include -iframework /Library/Developer/CommandLineTools/SDKs/MacOSX14.0.sdk/System/Library/Frameworks -fPIC -g -arch arm64 -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX14.0.sdk -fvisibility=hidden -stdlib=libc++  -Wall -Wextra -Wshadow -Wnon-virtual-dtor -Wcast-align -Wunused -Woverloaded-virtual -Wconversion -Wsign-conversion -Wdouble-promotion -Wformat=2 -Wnull-dereference -Wold-style-cast -Wpedantic -Werror -Wno-unknown-warning-option -MD -MT src/SFML/Window/CMakeFiles/sfml-window.dir/Cursor.cpp.o -MF src/SFML/Window/CMakeFiles/sfml-window.dir/Cursor.cpp.o.d -o src/SFML/Window/CMakeFiles/sfml-window.dir/Cursor.cpp.o -c /Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/src/SFML/Window/Cursor.cpp
[15/124] /Library/Developer/CommandLineTools/usr/bin/c++ -DSFML_STATIC -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/include -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/src -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/extlibs/headers/vulkan -isystem /Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/extlibs/headers/glad/include -iframework /Library/Developer/CommandLineTools/SDKs/MacOSX14.0.sdk/System/Library/Frameworks -fPIC -g -arch arm64 -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX14.0.sdk -fvisibility=hidden -stdlib=libc++  -Wall -Wextra -Wshadow -Wnon-virtual-dtor -Wcast-align -Wunused -Woverloaded-virtual -Wconversion -Wsign-conversion -Wdouble-promotion -Wformat=2 -Wnull-dereference -Wold-style-cast -Wpedantic -Werror -Wno-unknown-warning-option -MD -MT src/SFML/Window/CMakeFiles/sfml-window.dir/GlResource.cpp.o -MF src/SFML/Window/CMakeFiles/sfml-window.dir/GlResource.cpp.o.d -o src/SFML/Window/CMakeFiles/sfml-window.dir/GlResource.cpp.o -c /Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/src/SFML/Window/GlResource.cpp
[16/124] /Library/Developer/CommandLineTools/usr/bin/c++ -DSFML_STATIC -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/include -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/src -fPIC -g -arch arm64 -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX14.0.sdk -fvisibility=hidden -stdlib=libc++  -Wall -Wextra -Wshadow -Wnon-virtual-dtor -Wcast-align -Wunused -Woverloaded-virtual -Wconversion -Wsign-conversion -Wdouble-promotion -Wformat=2 -Wnull-dereference -Wold-style-cast -Wpedantic -Werror -Wno-unknown-warning-option -MD -MT src/SFML/System/CMakeFiles/sfml-system.dir/FileInputStream.cpp.o -MF src/SFML/System/CMakeFiles/sfml-system.dir/FileInputStream.cpp.o.d -o src/SFML/System/CMakeFiles/sfml-system.dir/FileInputStream.cpp.o -c /Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/src/SFML/System/FileInputStream.cpp
[17/124] /Library/Developer/CommandLineTools/usr/bin/c++ -DSFML_STATIC -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/include -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/src -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/extlibs/headers/vulkan -isystem /Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/extlibs/headers/glad/include -iframework /Library/Developer/CommandLineTools/SDKs/MacOSX14.0.sdk/System/Library/Frameworks -fPIC -g -arch arm64 -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX14.0.sdk -fvisibility=hidden -stdlib=libc++  -Wall -Wextra -Wshadow -Wnon-virtual-dtor -Wcast-align -Wunused -Woverloaded-virtual -Wconversion -Wsign-conversion -Wdouble-promotion -Wformat=2 -Wnull-dereference -Wold-style-cast -Wpedantic -Werror -Wno-unknown-warning-option -MD -MT src/SFML/Window/CMakeFiles/sfml-window.dir/Keyboard.cpp.o -MF src/SFML/Window/CMakeFiles/sfml-window.dir/Keyboard.cpp.o.d -o src/SFML/Window/CMakeFiles/sfml-window.dir/Keyboard.cpp.o -c /Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/src/SFML/Window/Keyboard.cpp
[18/124] /Library/Developer/CommandLineTools/usr/bin/c++ -DSFML_STATIC -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/include -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/src -fPIC -g -arch arm64 -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX14.0.sdk -fvisibility=hidden -stdlib=libc++  -Wall -Wextra -Wshadow -Wnon-virtual-dtor -Wcast-align -Wunused -Woverloaded-virtual -Wconversion -Wsign-conversion -Wdouble-promotion -Wformat=2 -Wnull-dereference -Wold-style-cast -Wpedantic -Werror -Wno-unknown-warning-option -MD -MT src/SFML/System/CMakeFiles/sfml-system.dir/Err.cpp.o -MF src/SFML/System/CMakeFiles/sfml-system.dir/Err.cpp.o.d -o src/SFML/System/CMakeFiles/sfml-system.dir/Err.cpp.o -c /Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/src/SFML/System/Err.cpp
[19/124] /Library/Developer/CommandLineTools/usr/bin/c++ -DSFML_STATIC -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/include -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/src -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/extlibs/headers/vulkan -isystem /Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/extlibs/headers/glad/include -iframework /Library/Developer/CommandLineTools/SDKs/MacOSX14.0.sdk/System/Library/Frameworks -fPIC -g -arch arm64 -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX14.0.sdk -fvisibility=hidden -stdlib=libc++  -Wall -Wextra -Wshadow -Wnon-virtual-dtor -Wcast-align -Wunused -Woverloaded-virtual -Wconversion -Wsign-conversion -Wdouble-promotion -Wformat=2 -Wnull-dereference -Wold-style-cast -Wpedantic -Werror -Wno-unknown-warning-option -MD -MT src/SFML/Window/CMakeFiles/sfml-window.dir/Touch.cpp.o -MF src/SFML/Window/CMakeFiles/sfml-window.dir/Touch.cpp.o.d -o src/SFML/Window/CMakeFiles/sfml-window.dir/Touch.cpp.o -c /Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/src/SFML/Window/Touch.cpp
[20/124] /Library/Developer/CommandLineTools/usr/bin/c++ -DSFML_STATIC -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/include -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/src -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/extlibs/headers/vulkan -isystem /Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/extlibs/headers/glad/include -iframework /Library/Developer/CommandLineTools/SDKs/MacOSX14.0.sdk/System/Library/Frameworks -fPIC -g -arch arm64 -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX14.0.sdk -fvisibility=hidden -stdlib=libc++  -Wall -Wextra -Wshadow -Wnon-virtual-dtor -Wcast-align -Wunused -Woverloaded-virtual -Wconversion -Wsign-conversion -Wdouble-promotion -Wformat=2 -Wnull-dereference -Wold-style-cast -Wpedantic -Werror -Wno-unknown-warning-option -MD -MT src/SFML/Window/CMakeFiles/sfml-window.dir/Sensor.cpp.o -MF src/SFML/Window/CMakeFiles/sfml-window.dir/Sensor.cpp.o.d -o src/SFML/Window/CMakeFiles/sfml-window.dir/Sensor.cpp.o -c /Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/src/SFML/Window/Sensor.cpp
[21/124] /Library/Developer/CommandLineTools/usr/bin/c++ -DSFML_STATIC -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/include -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/src -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/extlibs/headers/vulkan -isystem /Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/extlibs/headers/glad/include -iframework /Library/Developer/CommandLineTools/SDKs/MacOSX14.0.sdk/System/Library/Frameworks -fPIC -g -arch arm64 -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX14.0.sdk -fvisibility=hidden -stdlib=libc++  -Wall -Wextra -Wshadow -Wnon-virtual-dtor -Wcast-align -Wunused -Woverloaded-virtual -Wconversion -Wsign-conversion -Wdouble-promotion -Wformat=2 -Wnull-dereference -Wold-style-cast -Wpedantic -Werror -Wno-unknown-warning-option -MD -MT src/SFML/Window/CMakeFiles/sfml-window.dir/Clipboard.cpp.o -MF src/SFML/Window/CMakeFiles/sfml-window.dir/Clipboard.cpp.o.d -o src/SFML/Window/CMakeFiles/sfml-window.dir/Clipboard.cpp.o -c /Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/src/SFML/Window/Clipboard.cpp
[22/124] /Library/Developer/CommandLineTools/usr/bin/c++ -DSFML_STATIC -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/include -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/src -fPIC -g -arch arm64 -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX14.0.sdk -fvisibility=hidden -stdlib=libc++  -Wall -Wextra -Wshadow -Wnon-virtual-dtor -Wcast-align -Wunused -Woverloaded-virtual -Wconversion -Wsign-conversion -Wdouble-promotion -Wformat=2 -Wnull-dereference -Wold-style-cast -Wpedantic -Werror -Wno-unknown-warning-option -MD -MT src/SFML/System/CMakeFiles/sfml-system.dir/Unix/ThreadImpl.cpp.o -MF src/SFML/System/CMakeFiles/sfml-system.dir/Unix/ThreadImpl.cpp.o.d -o src/SFML/System/CMakeFiles/sfml-system.dir/Unix/ThreadImpl.cpp.o -c /Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/src/SFML/System/Unix/ThreadImpl.cpp
[23/124] /Library/Developer/CommandLineTools/usr/bin/c++ -DSFML_STATIC -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/include -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/src -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/extlibs/headers/vulkan -isystem /Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/extlibs/headers/glad/include -iframework /Library/Developer/CommandLineTools/SDKs/MacOSX14.0.sdk/System/Library/Frameworks -fPIC -g -arch arm64 -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX14.0.sdk -fvisibility=hidden -stdlib=libc++  -Wall -Wextra -Wshadow -Wnon-virtual-dtor -Wcast-align -Wunused -Woverloaded-virtual -Wconversion -Wsign-conversion -Wdouble-promotion -Wformat=2 -Wnull-dereference -Wold-style-cast -Wpedantic -Werror -Wno-unknown-warning-option -MD -MT src/SFML/Window/CMakeFiles/sfml-window.dir/Joystick.cpp.o -MF src/SFML/Window/CMakeFiles/sfml-window.dir/Joystick.cpp.o.d -o src/SFML/Window/CMakeFiles/sfml-window.dir/Joystick.cpp.o -c /Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/src/SFML/Window/Joystick.cpp
[24/124] /Library/Developer/CommandLineTools/usr/bin/c++ -DSFML_STATIC -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/include -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/src -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/extlibs/headers/vulkan -isystem /Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/extlibs/headers/glad/include -iframework /Library/Developer/CommandLineTools/SDKs/MacOSX14.0.sdk/System/Library/Frameworks -fPIC -g -arch arm64 -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX14.0.sdk -fvisibility=hidden -stdlib=libc++  -Wall -Wextra -Wshadow -Wnon-virtual-dtor -Wcast-align -Wunused -Woverloaded-virtual -Wconversion -Wsign-conversion -Wdouble-promotion -Wformat=2 -Wnull-dereference -Wold-style-cast -Wpedantic -Werror -Wno-unknown-warning-option -MD -MT src/SFML/Window/CMakeFiles/sfml-window.dir/JoystickManager.cpp.o -MF src/SFML/Window/CMakeFiles/sfml-window.dir/JoystickManager.cpp.o.d -o src/SFML/Window/CMakeFiles/sfml-window.dir/JoystickManager.cpp.o -c /Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/src/SFML/Window/JoystickManager.cpp
[25/124] /Library/Developer/CommandLineTools/usr/bin/c++ -DSFML_STATIC -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/include -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/src -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/extlibs/headers/vulkan -isystem /Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/extlibs/headers/glad/include -iframework /Library/Developer/CommandLineTools/SDKs/MacOSX14.0.sdk/System/Library/Frameworks -fPIC -g -arch arm64 -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX14.0.sdk -fvisibility=hidden -stdlib=libc++  -Wall -Wextra -Wshadow -Wnon-virtual-dtor -Wcast-align -Wunused -Woverloaded-virtual -Wconversion -Wsign-conversion -Wdouble-promotion -Wformat=2 -Wnull-dereference -Wold-style-cast -Wpedantic -Werror -Wno-unknown-warning-option -MD -MT src/SFML/Window/CMakeFiles/sfml-window.dir/Mouse.cpp.o -MF src/SFML/Window/CMakeFiles/sfml-window.dir/Mouse.cpp.o.d -o src/SFML/Window/CMakeFiles/sfml-window.dir/Mouse.cpp.o -c /Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/src/SFML/Window/Mouse.cpp
[26/124] /Library/Developer/CommandLineTools/usr/bin/c++ -DSFML_STATIC -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/include -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/src -fPIC -g -arch arm64 -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX14.0.sdk -fvisibility=hidden -stdlib=libc++  -Wall -Wextra -Wshadow -Wnon-virtual-dtor -Wcast-align -Wunused -Woverloaded-virtual -Wconversion -Wsign-conversion -Wdouble-promotion -Wformat=2 -Wnull-dereference -Wold-style-cast -Wpedantic -Werror -Wno-unknown-warning-option -MD -MT src/SFML/System/CMakeFiles/sfml-system.dir/String.cpp.o -MF src/SFML/System/CMakeFiles/sfml-system.dir/String.cpp.o.d -o src/SFML/System/CMakeFiles/sfml-system.dir/String.cpp.o -c /Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/src/SFML/System/String.cpp
[27/124] /Library/Developer/CommandLineTools/usr/bin/c++ -DSFML_STATIC -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/include -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/src -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/extlibs/headers/vulkan -isystem /Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/extlibs/headers/glad/include -iframework /Library/Developer/CommandLineTools/SDKs/MacOSX14.0.sdk/System/Library/Frameworks -fPIC -g -arch arm64 -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX14.0.sdk -fvisibility=hidden -stdlib=libc++  -Wall -Wextra -Wshadow -Wnon-virtual-dtor -Wcast-align -Wunused -Woverloaded-virtual -Wconversion -Wsign-conversion -Wdouble-promotion -Wformat=2 -Wnull-dereference -Wold-style-cast -Wpedantic -Werror -Wno-unknown-warning-option -MD -MT src/SFML/Window/CMakeFiles/sfml-window.dir/Vulkan.cpp.o -MF src/SFML/Window/CMakeFiles/sfml-window.dir/Vulkan.cpp.o.d -o src/SFML/Window/CMakeFiles/sfml-window.dir/Vulkan.cpp.o -c /Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/src/SFML/Window/Vulkan.cpp
[28/124] /Library/Developer/CommandLineTools/usr/bin/c++ -DSFML_STATIC -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/include -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/src -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/extlibs/headers/vulkan -isystem /Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/extlibs/headers/glad/include -iframework /Library/Developer/CommandLineTools/SDKs/MacOSX14.0.sdk/System/Library/Frameworks -fPIC -g -arch arm64 -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX14.0.sdk -fvisibility=hidden -stdlib=libc++  -Wall -Wextra -Wshadow -Wnon-virtual-dtor -Wcast-align -Wunused -Woverloaded-virtual -Wconversion -Wsign-conversion -Wdouble-promotion -Wformat=2 -Wnull-dereference -Wold-style-cast -Wpedantic -Werror -Wno-unknown-warning-option -MD -MT src/SFML/Window/CMakeFiles/sfml-window.dir/GlContext.cpp.o -MF src/SFML/Window/CMakeFiles/sfml-window.dir/GlContext.cpp.o.d -o src/SFML/Window/CMakeFiles/sfml-window.dir/GlContext.cpp.o -c /Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/src/SFML/Window/GlContext.cpp
[29/124] : && /opt/homebrew/Cellar/cmake/3.26.4/bin/cmake -E rm -f lib/libsfml-system-s-d.a && /Library/Developer/CommandLineTools/usr/bin/ar qc lib/libsfml-system-s-d.a  src/SFML/System/CMakeFiles/sfml-system.dir/Clock.cpp.o src/SFML/System/CMakeFiles/sfml-system.dir/Err.cpp.o src/SFML/System/CMakeFiles/sfml-system.dir/Lock.cpp.o src/SFML/System/CMakeFiles/sfml-system.dir/Mutex.cpp.o src/SFML/System/CMakeFiles/sfml-system.dir/Sleep.cpp.o src/SFML/System/CMakeFiles/sfml-system.dir/String.cpp.o src/SFML/System/CMakeFiles/sfml-system.dir/Thread.cpp.o src/SFML/System/CMakeFiles/sfml-system.dir/ThreadLocal.cpp.o src/SFML/System/CMakeFiles/sfml-system.dir/Time.cpp.o src/SFML/System/CMakeFiles/sfml-system.dir/FileInputStream.cpp.o src/SFML/System/CMakeFiles/sfml-system.dir/MemoryInputStream.cpp.o src/SFML/System/CMakeFiles/sfml-system.dir/Unix/ClockImpl.cpp.o src/SFML/System/CMakeFiles/sfml-system.dir/Unix/MutexImpl.cpp.o src/SFML/System/CMakeFiles/sfml-system.dir/Unix/SleepImpl.cpp.o src/SFML/System/CMakeFiles/sfml-system.dir/Unix/ThreadImpl.cpp.o src/SFML/System/CMakeFiles/sfml-system.dir/Unix/ThreadLocalImpl.cpp.o && /Library/Developer/CommandLineTools/usr/bin/ranlib lib/libsfml-system-s-d.a && /opt/homebrew/Cellar/cmake/3.26.4/bin/cmake -E touch lib/libsfml-system-s-d.a && :
[30/124] /Library/Developer/CommandLineTools/usr/bin/c++ -DSFML_STATIC -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/include -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/src -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/extlibs/headers/vulkan -isystem /Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/extlibs/headers/glad/include -iframework /Library/Developer/CommandLineTools/SDKs/MacOSX14.0.sdk/System/Library/Frameworks -fPIC -g -arch arm64 -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX14.0.sdk -fvisibility=hidden -stdlib=libc++  -Wall -Wextra -Wshadow -Wnon-virtual-dtor -Wcast-align -Wunused -Woverloaded-virtual -Wconversion -Wsign-conversion -Wdouble-promotion -Wformat=2 -Wnull-dereference -Wold-style-cast -Wpedantic -Werror -Wno-unknown-warning-option -MD -MT src/SFML/Window/CMakeFiles/sfml-window.dir/VideoMode.cpp.o -MF src/SFML/Window/CMakeFiles/sfml-window.dir/VideoMode.cpp.o.d -o src/SFML/Window/CMakeFiles/sfml-window.dir/VideoMode.cpp.o -c /Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/src/SFML/Window/VideoMode.cpp
[31/124] /Library/Developer/CommandLineTools/usr/bin/c++ -DSFML_STATIC -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/include -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/src -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/extlibs/headers/vulkan -isystem /Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/extlibs/headers/glad/include -iframework /Library/Developer/CommandLineTools/SDKs/MacOSX14.0.sdk/System/Library/Frameworks -fPIC -g -arch arm64 -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX14.0.sdk -fvisibility=hidden -stdlib=libc++  -Wall -Wextra -Wshadow -Wnon-virtual-dtor -Wcast-align -Wunused -Woverloaded-virtual -Wconversion -Wsign-conversion -Wdouble-promotion -Wformat=2 -Wnull-dereference -Wold-style-cast -Wpedantic -Werror -Wno-unknown-warning-option -MD -MT src/SFML/Window/CMakeFiles/sfml-window.dir/SensorManager.cpp.o -MF src/SFML/Window/CMakeFiles/sfml-window.dir/SensorManager.cpp.o.d -o src/SFML/Window/CMakeFiles/sfml-window.dir/SensorManager.cpp.o -c /Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/src/SFML/Window/SensorManager.cpp
[32/124] /Library/Developer/CommandLineTools/usr/bin/c++ -DSFML_STATIC -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/include -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/src -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/extlibs/headers/vulkan -isystem /Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/extlibs/headers/glad/include -iframework /Library/Developer/CommandLineTools/SDKs/MacOSX14.0.sdk/System/Library/Frameworks -fPIC -g -arch arm64 -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX14.0.sdk -fvisibility=hidden -stdlib=libc++  -Wall -Wextra -Wshadow -Wnon-virtual-dtor -Wcast-align -Wunused -Woverloaded-virtual -Wconversion -Wsign-conversion -Wdouble-promotion -Wformat=2 -Wnull-dereference -Wold-style-cast -Wpedantic -Werror -Wno-unknown-warning-option -MD -MT src/SFML/Window/CMakeFiles/sfml-window.dir/Window.cpp.o -MF src/SFML/Window/CMakeFiles/sfml-window.dir/Window.cpp.o.d -o src/SFML/Window/CMakeFiles/sfml-window.dir/Window.cpp.o -c /Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/src/SFML/Window/Window.cpp
[33/124] /Library/Developer/CommandLineTools/usr/bin/c++ -DSFML_STATIC -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/include -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/src -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/extlibs/headers/vulkan -isystem /Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/extlibs/headers/glad/include -iframework /Library/Developer/CommandLineTools/SDKs/MacOSX14.0.sdk/System/Library/Frameworks -fPIC -g -arch arm64 -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX14.0.sdk -fvisibility=hidden -stdlib=libc++  -Wall -Wextra -Wshadow -Wnon-virtual-dtor -Wcast-align -Wunused -Woverloaded-virtual -Wconversion -Wsign-conversion -Wdouble-promotion -Wformat=2 -Wnull-dereference -Wold-style-cast -Wpedantic -Werror -Wno-unknown-warning-option -MD -MT src/SFML/Window/CMakeFiles/sfml-window.dir/WindowBase.cpp.o -MF src/SFML/Window/CMakeFiles/sfml-window.dir/WindowBase.cpp.o.d -o src/SFML/Window/CMakeFiles/sfml-window.dir/WindowBase.cpp.o -c /Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/src/SFML/Window/WindowBase.cpp
[34/124] /Library/Developer/CommandLineTools/usr/bin/c++ -DSFML_STATIC -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/include -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/src -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/extlibs/headers/vulkan -isystem /Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/extlibs/headers/glad/include -iframework /Library/Developer/CommandLineTools/SDKs/MacOSX14.0.sdk/System/Library/Frameworks -fPIC -g -arch arm64 -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX14.0.sdk -fvisibility=hidden -stdlib=libc++  -Wall -Wextra -Wshadow -Wnon-virtual-dtor -Wcast-align -Wunused -Woverloaded-virtual -Wconversion -Wsign-conversion -Wdouble-promotion -Wformat=2 -Wnull-dereference -Wold-style-cast -Wpedantic -Werror -Wno-unknown-warning-option -MD -MT src/SFML/Window/CMakeFiles/sfml-window.dir/OSX/CursorImpl.mm.o -MF src/SFML/Window/CMakeFiles/sfml-window.dir/OSX/CursorImpl.mm.o.d -o src/SFML/Window/CMakeFiles/sfml-window.dir/OSX/CursorImpl.mm.o -c /Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/src/SFML/Window/OSX/CursorImpl.mm
[35/124] /Library/Developer/CommandLineTools/usr/bin/c++ -DSFML_STATIC -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/include -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/src -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/extlibs/headers/vulkan -isystem /Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/extlibs/headers/glad/include -iframework /Library/Developer/CommandLineTools/SDKs/MacOSX14.0.sdk/System/Library/Frameworks -fPIC -g -arch arm64 -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX14.0.sdk -fvisibility=hidden -stdlib=libc++  -Wall -Wextra -Wshadow -Wnon-virtual-dtor -Wcast-align -Wunused -Woverloaded-virtual -Wconversion -Wsign-conversion -Wdouble-promotion -Wformat=2 -Wnull-dereference -Wold-style-cast -Wpedantic -Werror -Wno-unknown-warning-option -MD -MT src/SFML/Window/CMakeFiles/sfml-window.dir/WindowImpl.cpp.o -MF src/SFML/Window/CMakeFiles/sfml-window.dir/WindowImpl.cpp.o.d -o src/SFML/Window/CMakeFiles/sfml-window.dir/WindowImpl.cpp.o -c /Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/src/SFML/Window/WindowImpl.cpp
[36/124] /Library/Developer/CommandLineTools/usr/bin/c++ -DSFML_STATIC -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/include -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/src -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/extlibs/headers/vulkan -isystem /Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/extlibs/headers/glad/include -iframework /Library/Developer/CommandLineTools/SDKs/MacOSX14.0.sdk/System/Library/Frameworks -fPIC -g -arch arm64 -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX14.0.sdk -fvisibility=hidden -stdlib=libc++  -Wall -Wextra -Wshadow -Wnon-virtual-dtor -Wcast-align -Wunused -Woverloaded-virtual -Wconversion -Wsign-conversion -Wdouble-promotion -Wformat=2 -Wnull-dereference -Wold-style-cast -Wpedantic -Werror -Wno-unknown-warning-option -MD -MT src/SFML/Window/CMakeFiles/sfml-window.dir/OSX/SensorImpl.cpp.o -MF src/SFML/Window/CMakeFiles/sfml-window.dir/OSX/SensorImpl.cpp.o.d -o src/SFML/Window/CMakeFiles/sfml-window.dir/OSX/SensorImpl.cpp.o -c /Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/src/SFML/Window/OSX/SensorImpl.cpp
[37/124] /Library/Developer/CommandLineTools/usr/bin/cc -DSFML_STATIC -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/include -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/src -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/extlibs/headers/vulkan -isystem /Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/extlibs/headers/glad/include -iframework /Library/Developer/CommandLineTools/SDKs/MacOSX14.0.sdk/System/Library/Frameworks -fPIC -g -arch arm64 -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX14.0.sdk -fvisibility=hidden -stdlib=libc++  -Wall -Wextra -Wshadow -Wnon-virtual-dtor -Wcast-align -Wunused -Woverloaded-virtual -Wconversion -Wsign-conversion -Wdouble-promotion -Wformat=2 -Wnull-dereference -Wold-style-cast -Wpedantic -Werror -Wno-unknown-warning-option -MD -MT src/SFML/Window/CMakeFiles/sfml-window.dir/OSX/SFApplication.m.o -MF src/SFML/Window/CMakeFiles/sfml-window.dir/OSX/SFApplication.m.o.d -o src/SFML/Window/CMakeFiles/sfml-window.dir/OSX/SFApplication.m.o -c /Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/src/SFML/Window/OSX/SFApplication.m
FAILED: src/SFML/Window/CMakeFiles/sfml-window.dir/OSX/SFApplication.m.o 
/Library/Developer/CommandLineTools/usr/bin/cc -DSFML_STATIC -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/include -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/src -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/extlibs/headers/vulkan -isystem /Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/extlibs/headers/glad/include -iframework /Library/Developer/CommandLineTools/SDKs/MacOSX14.0.sdk/System/Library/Frameworks -fPIC -g -arch arm64 -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX14.0.sdk -fvisibility=hidden -stdlib=libc++  -Wall -Wextra -Wshadow -Wnon-virtual-dtor -Wcast-align -Wunused -Woverloaded-virtual -Wconversion -Wsign-conversion -Wdouble-promotion -Wformat=2 -Wnull-dereference -Wold-style-cast -Wpedantic -Werror -Wno-unknown-warning-option -MD -MT src/SFML/Window/CMakeFiles/sfml-window.dir/OSX/SFApplication.m.o -MF src/SFML/Window/CMakeFiles/sfml-window.dir/OSX/SFApplication.m.o.d -o src/SFML/Window/CMakeFiles/sfml-window.dir/OSX/SFApplication.m.o -c /Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/src/SFML/Window/OSX/SFApplication.m
clang: error: argument unused during compilation: '-stdlib=libc++' [-Werror,-Wunused-command-line-argument]
[38/124] /Library/Developer/CommandLineTools/usr/bin/c++ -DSFML_STATIC -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/include -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/src -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/extlibs/headers/vulkan -isystem /Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/extlibs/headers/glad/include -iframework /Library/Developer/CommandLineTools/SDKs/MacOSX14.0.sdk/System/Library/Frameworks -fPIC -g -arch arm64 -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX14.0.sdk -fvisibility=hidden -stdlib=libc++  -Wall -Wextra -Wshadow -Wnon-virtual-dtor -Wcast-align -Wunused -Woverloaded-virtual -Wconversion -Wsign-conversion -Wdouble-promotion -Wformat=2 -Wnull-dereference -Wold-style-cast -Wpedantic -Werror -Wno-unknown-warning-option -MD -MT src/SFML/Window/CMakeFiles/sfml-window.dir/OSX/cpp_objc_conversion.mm.o -MF src/SFML/Window/CMakeFiles/sfml-window.dir/OSX/cpp_objc_conversion.mm.o.d -o src/SFML/Window/CMakeFiles/sfml-window.dir/OSX/cpp_objc_conversion.mm.o -c /Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/src/SFML/Window/OSX/cpp_objc_conversion.mm
[39/124] /Library/Developer/CommandLineTools/usr/bin/c++ -DSFML_STATIC -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/include -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/src -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/extlibs/headers/vulkan -isystem /Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/extlibs/headers/glad/include -iframework /Library/Developer/CommandLineTools/SDKs/MacOSX14.0.sdk/System/Library/Frameworks -fPIC -g -arch arm64 -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX14.0.sdk -fvisibility=hidden -stdlib=libc++  -Wall -Wextra -Wshadow -Wnon-virtual-dtor -Wcast-align -Wunused -Woverloaded-virtual -Wconversion -Wsign-conversion -Wdouble-promotion -Wformat=2 -Wnull-dereference -Wold-style-cast -Wpedantic -Werror -Wno-unknown-warning-option -MD -MT src/SFML/Window/CMakeFiles/sfml-window.dir/OSX/cg_sf_conversion.mm.o -MF src/SFML/Window/CMakeFiles/sfml-window.dir/OSX/cg_sf_conversion.mm.o.d -o src/SFML/Window/CMakeFiles/sfml-window.dir/OSX/cg_sf_conversion.mm.o -c /Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/src/SFML/Window/OSX/cg_sf_conversion.mm
[40/124] /Library/Developer/CommandLineTools/usr/bin/c++ -DSFML_STATIC -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/include -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/src -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/extlibs/headers/vulkan -isystem /Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/extlibs/headers/glad/include -iframework /Library/Developer/CommandLineTools/SDKs/MacOSX14.0.sdk/System/Library/Frameworks -fPIC -g -arch arm64 -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX14.0.sdk -fvisibility=hidden -stdlib=libc++  -Wall -Wextra -Wshadow -Wnon-virtual-dtor -Wcast-align -Wunused -Woverloaded-virtual -Wconversion -Wsign-conversion -Wdouble-promotion -Wformat=2 -Wnull-dereference -Wold-style-cast -Wpedantic -Werror -Wno-unknown-warning-option -MD -MT src/SFML/Window/CMakeFiles/sfml-window.dir/OSX/ClipboardImpl.mm.o -MF src/SFML/Window/CMakeFiles/sfml-window.dir/OSX/ClipboardImpl.mm.o.d -o src/SFML/Window/CMakeFiles/sfml-window.dir/OSX/ClipboardImpl.mm.o -c /Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/src/SFML/Window/OSX/ClipboardImpl.mm
[41/124] /Library/Developer/CommandLineTools/usr/bin/c++ -DSFML_STATIC -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/include -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/src -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/extlibs/headers/vulkan -isystem /Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/extlibs/headers/glad/include -iframework /Library/Developer/CommandLineTools/SDKs/MacOSX14.0.sdk/System/Library/Frameworks -fPIC -g -arch arm64 -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX14.0.sdk -fvisibility=hidden -stdlib=libc++  -Wall -Wextra -Wshadow -Wnon-virtual-dtor -Wcast-align -Wunused -Woverloaded-virtual -Wconversion -Wsign-conversion -Wdouble-promotion -Wformat=2 -Wnull-dereference -Wold-style-cast -Wpedantic -Werror -Wno-unknown-warning-option -MD -MT src/SFML/Window/CMakeFiles/sfml-window.dir/OSX/HIDJoystickManager.cpp.o -MF src/SFML/Window/CMakeFiles/sfml-window.dir/OSX/HIDJoystickManager.cpp.o.d -o src/SFML/Window/CMakeFiles/sfml-window.dir/OSX/HIDJoystickManager.cpp.o -c /Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/src/SFML/Window/OSX/HIDJoystickManager.cpp
[42/124] /Library/Developer/CommandLineTools/usr/bin/c++ -DSFML_STATIC -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/include -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/src -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/extlibs/headers/vulkan -isystem /Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/extlibs/headers/glad/include -iframework /Library/Developer/CommandLineTools/SDKs/MacOSX14.0.sdk/System/Library/Frameworks -fPIC -g -arch arm64 -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX14.0.sdk -fvisibility=hidden -stdlib=libc++  -Wall -Wextra -Wshadow -Wnon-virtual-dtor -Wcast-align -Wunused -Woverloaded-virtual -Wconversion -Wsign-conversion -Wdouble-promotion -Wformat=2 -Wnull-dereference -Wold-style-cast -Wpedantic -Werror -Wno-unknown-warning-option -MD -MT src/SFML/Window/CMakeFiles/sfml-window.dir/OSX/NSImage+raw.mm.o -MF src/SFML/Window/CMakeFiles/sfml-window.dir/OSX/NSImage+raw.mm.o.d -o src/SFML/Window/CMakeFiles/sfml-window.dir/OSX/NSImage+raw.mm.o -c /Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/src/SFML/Window/OSX/NSImage+raw.mm
[43/124] /Library/Developer/CommandLineTools/usr/bin/c++ -DSFML_STATIC -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/include -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/src -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/extlibs/headers/vulkan -isystem /Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/extlibs/headers/glad/include -iframework /Library/Developer/CommandLineTools/SDKs/MacOSX14.0.sdk/System/Library/Frameworks -fPIC -g -arch arm64 -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX14.0.sdk -fvisibility=hidden -stdlib=libc++  -Wall -Wextra -Wshadow -Wnon-virtual-dtor -Wcast-align -Wunused -Woverloaded-virtual -Wconversion -Wsign-conversion -Wdouble-promotion -Wformat=2 -Wnull-dereference -Wold-style-cast -Wpedantic -Werror -Wno-unknown-warning-option -MD -MT src/SFML/Window/CMakeFiles/sfml-window.dir/OSX/InputImpl.mm.o -MF src/SFML/Window/CMakeFiles/sfml-window.dir/OSX/InputImpl.mm.o.d -o src/SFML/Window/CMakeFiles/sfml-window.dir/OSX/InputImpl.mm.o -c /Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/src/SFML/Window/OSX/InputImpl.mm
[44/124] /Library/Developer/CommandLineTools/usr/bin/c++ -DSFML_STATIC -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/include -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/src -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/extlibs/headers/vulkan -isystem /Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/extlibs/headers/glad/include -iframework /Library/Developer/CommandLineTools/SDKs/MacOSX14.0.sdk/System/Library/Frameworks -fPIC -g -arch arm64 -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX14.0.sdk -fvisibility=hidden -stdlib=libc++  -Wall -Wextra -Wshadow -Wnon-virtual-dtor -Wcast-align -Wunused -Woverloaded-virtual -Wconversion -Wsign-conversion -Wdouble-promotion -Wformat=2 -Wnull-dereference -Wold-style-cast -Wpedantic -Werror -Wno-unknown-warning-option -MD -MT src/SFML/Window/CMakeFiles/sfml-window.dir/OSX/JoystickImpl.cpp.o -MF src/SFML/Window/CMakeFiles/sfml-window.dir/OSX/JoystickImpl.cpp.o.d -o src/SFML/Window/CMakeFiles/sfml-window.dir/OSX/JoystickImpl.cpp.o -c /Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/src/SFML/Window/OSX/JoystickImpl.cpp
[45/124] /Library/Developer/CommandLineTools/usr/bin/c++ -DSFML_STATIC -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/include -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/src -I/Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/extlibs/headers/vulkan -isystem /Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/extlibs/headers/glad/include -iframework /Library/Developer/CommandLineTools/SDKs/MacOSX14.0.sdk/System/Library/Frameworks -fPIC -g -arch arm64 -isysroot /Library/Developer/CommandLineTools/SDKs/MacOSX14.0.sdk -fvisibility=hidden -stdlib=libc++  -Wall -Wextra -Wshadow -Wnon-virtual-dtor -Wcast-align -Wunused -Woverloaded-virtual -Wconversion -Wsign-conversion -Wdouble-promotion -Wformat=2 -Wnull-dereference -Wold-style-cast -Wpedantic -Werror -Wno-unknown-warning-option -MD -MT src/SFML/Window/CMakeFiles/sfml-window.dir/OSX/HIDInputManager.mm.o -MF src/SFML/Window/CMakeFiles/sfml-window.dir/OSX/HIDInputManager.mm.o.d -o src/SFML/Window/CMakeFiles/sfml-window.dir/OSX/HIDInputManager.mm.o -c /Users/pierre/Code/VectorAudio/vcpkg/buildtrees/sfml/src/2.6.0-2d6a42d05d.clean/src/SFML/Window/OSX/HIDInputManager.mm
ninja: build stopped: subcommand failed.
```
</details>

**Additional context**

<details><summary>vcpkg.json</summary>

```
{
  "$schema": "https://raw.githubusercontent.com/microsoft/vcpkg-tool/main/docs/vcpkg.schema.json",
  "name": "vector-audio",
  "version": "1.1.0",
  "dependencies": [
    "nlohmann-json",
    "toml11",
    "cpp-httplib",
    "curl",
    "fmt",
    "spdlog",
    "restinio",
    "opus",
    "speexdsp",
    "cpp-jwt",
    "msgpack",
    "libevent",
    "neargye-semver",
    "sfml"
  ]
}

```
</details>

--- a/ports/sfml/portfile.cmake
+++ b/ports/sfml/portfile.cmake
@@ -21,7 +21,7 @@ vcpkg_cmake_configure(
         -DSFML_USE_SYSTEM_DEPS=ON
         -DSFML_MISC_INSTALL_PREFIX=share/sfml
         -DSFML_GENERATE_PDB=OFF
-		-DSFML_WARNINGS_AS_ERRORS=OFF #Remove in the next version
+        -DSFML_WARNINGS_AS_ERRORS=OFF #Remove in the next version
 )
 
 vcpkg_cmake_install()

--- a/ports/sfml/portfile.cmake
+++ b/ports/sfml/portfile.cmake
@@ -21,6 +21,7 @@ vcpkg_cmake_configure(
         -DSFML_USE_SYSTEM_DEPS=ON
         -DSFML_MISC_INSTALL_PREFIX=share/sfml
         -DSFML_GENERATE_PDB=OFF
+		-DSFML_WARNINGS_AS_ERRORS=OFF #Remove in the next version
 )
 
 vcpkg_cmake_install()

--- a/ports/sfml/vcpkg.json
+++ b/ports/sfml/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "sfml",
   "version": "2.6.0",
-  "port-version": 17,
+  "port-version": 18,
   "description": "Simple and fast multimedia library",
   "homepage": "https://github.com/SFML/SFML",
   "license": "Zlib",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -7514,7 +7514,7 @@
     },
     "sfml": {
       "baseline": "2.6.0",
-      "port-version": 17
+      "port-version": 18
     },
     "sfsexp": {
       "baseline": "1.3.1",

--- a/versions/s-/sfml.json
+++ b/versions/s-/sfml.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "73e4e0a33324f5962b47c21ce9e337799e1a3800",
+      "version": "2.6.0",
+      "port-version": 18
+    },
+    {
       "git-tree": "8d61263f546c87d3c26f45ade4f92d65f1203816",
       "version": "2.6.0",
       "port-version": 17

--- a/versions/s-/sfml.json
+++ b/versions/s-/sfml.json
@@ -1,7 +1,7 @@
 {
   "versions": [
     {
-      "git-tree": "73e4e0a33324f5962b47c21ce9e337799e1a3800",
+      "git-tree": "7712e31893217c566173961ad3681c19856e69e6",
       "version": "2.6.0",
       "port-version": 18
     },

